src/test/resources/reporting/datamodelbase-testing-feature.ttl

```diff
@@ -10,7 +10,7 @@ ex:test
     ex:hasInt 1 ;
     ex:hasBool true ;
```

Comment:
I don't quite understand these..

Reply:
Just testing how data is loaded into a Java bean